mlir/test/Quantum/ScatterTest.mlir

```diff
@@ -45,13 +45,18 @@ func.func public @scatter_multiply(%arg0: tensor<3xf64>, %arg1: tensor<i64>) ->
     // CHECK-NEXT:   return [[RESTENSOR]] : tensor<f64>
 
 // CHECK: func.func public @scatter_multiply(%arg0: tensor<3xf64>, %arg1: tensor<i64>) -> tensor<3xf64>
-    // CHECK:   [[CST:%.+]] = arith.constant dense<2.000000e+00> : tensor<f64>
-    // CHECK:   [[INPUTEXTRACTED:%.+]] = tensor.extract %arg0
-    // CHECK:   [[INPUTEXTRACTED1:%.+]] = tensor.from_elements [[INPUTEXTRACTED]] : tensor<f64>
-    // CHECK:   [[CALL:%.+]] = call @__catalyst_update_scatter[[NUMBER]]([[INPUTEXTRACTED1]], [[CST]]) : (tensor<f64>, tensor<f64>) -> tensor<f64>
-    // CHECK:   [[RESEXTRACTED:%.+]] = tensor.extract [[CALL]][] : tensor<f64>
-    // CHECK:   [[INSERT:%.+]] = tensor.insert [[RESEXTRACTED]] into %arg0[%4] : tensor<3xf64>
-    // CHECK:   return [[INSERT]] : tensor<3xf64>
+    // CHECK:    [[IDX0:%.+]] = index.constant 0
+    // CHECK:    [[IDX1:%.+]] = index.constant 1
+    // CHECK:    [[CST:%.+]] = arith.constant dense<2.000000e+00> : tensor<f64>
+    // CHECK:    [[SCF:%.+]] = scf.for %arg2 = [[IDX0]] to [[IDX1]] step [[IDX1]] iter_args(%arg3 = %arg0) -> (tensor<3xf64>)
+    // CHECK:    [[INDEX:%.+]] = arith.index_cast %6 : i32 to index
```

Comment:
`%6` shouldn't exist in the test, what value is it referring to?

Reply:
I also remove it like suggested